src/lib/lessons.ts

```diff
@@ -4,23 +4,47 @@ import getAccessTokenFromCookie from '../utils/get-access-token-from-cookie'
 import {loadLessonComments} from './lesson-comments'
 import {sanityClient} from 'utils/sanity-client'
 import groq from 'groq'
+import isEmpty from 'lodash/isEmpty'
+import {mergeLessonMetadata} from 'utils/lesson-metadata'
 
+// code_url is only used in a select few Kent C. Dodds lessons
 const lessonQuery = groq`
-*[_type == 'lesson' && slug == $slug][0]{
+*[_type == 'lesson' && slug.current == $slug][0]{
   title,
-  slug,
+  'slug': slug.current,
   description,
+  resource->_type == 'videoResource' => {
+    ...(resource-> {
+      'media_url': hslUrl,
+      'transcript': transcriptBody,
+      'transcript_url': transcriptUrl,
+      duration,
+      'subtitles_url': subtitlesUrl,
+    })
+  },
+  'free_forever': isCommunityResource,
+  'path': '/lessons/' + slug.current,
+  'thumb_url': thumbnailUrl,
+  'repo_url': repoUrl,
+  'code_url': codeUrl,
+  'createdAt': eggheadRailsCreatedAt,
+  'updatedAt': eggheadRailsUpdatedAt,
+  'publishedAt': eggheadRailsPublishedAt,
```

Comment:
TODO: we accidentally reverted the camel case on these three, so they aren't going to override the values in the graphql query.

Reply:
fixed here: https://github.com/eggheadio/egghead-next/pull/1154/commits/847f1dc537ac08adda6c81f7a0f5a0fa28dd61a6